### DISCUSSION TITLE
Doubt about Wordpress website and website coding

### DISCUSSION THREAD

**Discussion by Nemesi965** [community/community#31809] [Codespaces]
Hi guys, I have a doubt: is it possible to create a website with in html / css / js and then insert it into a clean installation of wordpress? Would it still be possible to use Wordpress plugins? I ask because I have heard some developers do such a thing, but I never understood, if you create a site in html / css / js code you must then insert it necessarily within a CMS platform or simply insert it inside the directory of the Which hosting service do you use for your site to make it work?

**Comment by jfullstackdev** [comment]
what you are referring to is maybe this one 

https://wpbuffs.com/edit-source-code-in-wordpress/

that you can directly edit the code on WordPress, that's actually advantageous
